docs/zh_CN/minio-admin-complete-guide.md

```diff
@@ -0,0 +1,468 @@
+# MinIO 管理完全指南 [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io)
+
+MinIO Client (mc) 提供了`admin`子命令以在MinIO部署上执行管理任务.
+
+```sh
+service      停止，重启或得到MinIO服务器的状态
+info         显示MinIO服务器信息
```

Comment:
Per Google translate 
```
User management user
Policy management encapsulation policy
Config management profile
```
Not sure if this is the correct syntax or not, should it translate as "manage" instead of "management"?  Do we have someone who can verify?

Reply:
The word "management" and "manage" are exactly the same when translated into Chinese. Maybe that's why google made a wrong translation. And I've found that google will translate it into "management" when the input sentence lacks subject even though it is a verb and translate it into "manage" when it has. So don't worry about that.